pkg/cache/v3/delta_test.go

```diff
@@ -24,9 +24,7 @@ import (
 func assertResourceMapEqual(t *testing.T, want, got map[string]types.Resource) {
 	t.Helper()
 
-	if !cmp.Equal(want, got, protocmp.Transform()) {
-		t.Errorf("got resources %v, want %v", got, want)
-	}
+	assert.Truef(t, cmp.Equal(want, got, protocmp.Transform()), "got resources %v, want %v", got, want)
```

Comment:
Nit, using `cmp.Diff` in the error message makes it much simpler to process errors imo